src/library_webgpu.js

```diff
@@ -1317,9 +1312,10 @@ var LibraryWebGPU = {
 
     function makeVertexBuffer(vbPtr) {
       if (!vbPtr) return undefined;
-      return {
+      var stepModeValue = WebGPU.VertexStepMode[{{{ gpu.makeGetU32('vbPtr', C_STRUCTS.WGPUVertexBufferLayout.stepMode) }}}];
+      return stepModeValue === undefined ? undefined :{
```

Comment:
nit: instead of checking `stepModeValue === undefined` it makes more sense to check `stepModeInt === {{{ VertexBufferNotUsed }}}`, because the `undefined` doesn't actually mean anything useful in JS.